packages/atomic/src/components/result-lists/list-display-results.tsx

```diff
@@ -12,16 +12,14 @@ export const ListDisplayResults: FunctionalComponent<ResultsProps> = (
         result={result}
         engine={props.bindings.engine}
         store={props.bindings.store}
-        display={props.display}
-        density={props.density}
-        imageSize={props.imageSize}
         content={props.getContentOfResultTemplate(result)}
         loadingFlag={props.resultListCommon.loadingFlag}
         ref={(element) =>
           element &&
           props.indexOfResultToFocus === index &&
           props.newResultRef?.(element)
         }
+        {...props}
```

Comment:
Yeah that's fine, since the types expected by `atomic-result-result` are strict. 

For example, if we wanted to use `newResultRef` inside `atomic-result`, we'd have to explicitly add it to the props of `atomic-result` (or some weird cast to any), or it's otherwise not accessible.